File: nodejs/status.js
const app = new (require('express').Router)()

app.get('/paybear/status/:order', (req, res) => {
  // const orderId = req.params.order
  const confirmations = 0 // get from DB, see callback.php
  const maxConfirmations = 3 // get from DB, see callback.php
  const resp = {
    success: confirmations >= maxConfirmations
  }
  if (confirmations !== null) { resp.confirmations = confirmations }
  res.json(resp) // return this data to PayBear form
})

module.exports = app

File: rails/status.rb
def status
  @MIN_CONFIRMATIONS = 3

  if params[:order]
    orderId = params[:order]

    confirmations = nil

    confirmations = 0 # get from DB, see callback.rb
    max_confirmations = 3 # get from DB, see callback.rb

    data = {
      success: confirmations >= max_confirmations
    }

    if confirmations
      data['confirmations'] = confirmations
    end

    render json: data
  end
end